MvcMusicStore/Web.Release.config

```diff
@@ -3,17 +3,15 @@
 <!-- For more information on using web.config transformation visit http://go.microsoft.com/fwlink/?LinkId=125889 -->
 
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
-  <!--
-    In the example below, the "SetAttributes" transform will change the value of 
-    "connectionString" to use "ReleaseSQLServer" only when the "Match" locator 
-    finds an atrribute "name" that has a value of "MyDB".
-    
-    <connectionStrings>
-      <add name="MyDB" 
-        connectionString="Data Source=ReleaseSQLServer;Initial Catalog=MyReleaseDB;Integrated Security=True" 
-        xdt:Transform="SetAttributes" xdt:Locator="Match(name)"/>
-    </connectionStrings>
-  -->
+  <connectionStrings>
+    <add name="ASPState" connectionString="Data Source=<<YOUR RDS INSTANCE ENDPOINT>>;Integrated Security=SSPI" providerName="System.Data.SqlClient"
```

Comment:
@sagulati Do we use config builder to override connection strings from environment variables at run time?

Reply:
good point Vlad, We don't have config builder in this branch but would be a good add to allow connection string override.